.github/workflows/main.yml

```diff
@@ -62,7 +65,7 @@ jobs:
     - name: Test release
       run: |
         # verify that version information is embedded correctly
-        ./dist/chezmoi-cgo_linux_amd64/chezmoi --version | tee /dev/stderr | grep -q "chezmoi version v"
+        file ./dist/chezmoi-cgo_linux_amd64/chezmoi --version | tee /dev/stderr | grep -q "statically linked"
```

Comment:
```suggestion
        file ./dist/chezmoi-cgo_linux_amd64/chezmoi | tee /dev/stderr | grep -q "statically linked"
```

Reply:
Good catch, thanks! I was a bit sloppy with cut-and-paste.